carbonmark/lib/retirementDataQueries/retirementDataViaPolygonDigitalCarbon.ts

```diff
@@ -121,15 +121,15 @@ export const queryKlimaRetireByIndex = async (
 
 export const queryKlimaRetiresByAddress = async (
   beneficiaryAddress: string
-): Promise<KlimaRetire[] | false> => {
+): Promise<KlimaRetire[]> => {
   try {
     const json: QueryKlimaRetires = await fetchGraphQL(
       generateKlimaRetireQuery(beneficiaryAddress)
     );
-    return !!json.data.klimaRetires.length && json.data.klimaRetires;
+    return json.data.klimaRetires || [];
   } catch (e) {
     console.error("Failed to query KlimaRetiresByAddress", e);
-    return Promise.reject(e);
+    return [];
```

Comment:
If we're not going to be rejecting the promise we probably need to be throwing an error here instead.

If it's expected behaviour "sometimes" then we should be using `console.info`